tldw_Server_API/tests/MCP_unified/test_tool_catalogs_api.py

```
@@ -73,7 +73,7 @@ def _default_for_type(col_type: str):
 
             placeholders = ", ".join("?" for _ in insert_cols)
             conn.execute(
-                f"INSERT INTO users ({', '.join(insert_cols)}) VALUES ({placeholders})",
+                f"INSERT INTO users ({', '.join(insert_cols)}) VALUES ({placeholders})",  # nosec B608
```

Ruff (0.15.1)

[error] Possible SQL injection vector through string-based query construction

(S608)